# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/MediaManager.h
# Commit: 98f0dd0b45cf
# Full Hash: 98f0dd0b45cf207837b59909bec2e253af15fecd
# Author: Paul Adenot <paul@paul.cx>
# Date: 2020-03-28 09:32:56
# Regressor Bug: 1624322
# File Overlap Count: 1
# Description:
#   Bug 1624322 - Add the raw group ID to MediaDevice. r=jib
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D68082
# ==============================================================================

diff -r f3a28fba69d4 -r 98f0dd0b45cf dom/media/MediaManager.h
--- a/dom/media/MediaManager.h	Fri Mar 27 17:57:48 2020 +0000
+++ b/dom/media/MediaManager.h	Fri Mar 27 18:40:57 2020 +0000
@@ -75,11 +75,12 @@
               const nsString& aRawID = NS_LITERAL_STRING(""));
 
   MediaDevice(const RefPtr<MediaDevice>& aOther, const nsString& aID,
-              const nsString& aGroupID, const nsString& aRawID);
+              const nsString& aGroupID, const nsString& aRawID,
+              const nsString& aRawGroupID);
 
   MediaDevice(const RefPtr<MediaDevice>& aOther, const nsString& aID,
               const nsString& aGroupID, const nsString& aRawID,
-              const nsString& aName);
+              const nsString& aRawGroupID, const nsString& aName);
 
   uint32_t GetBestFitnessDistance(
       const nsTArray<const NormalizedConstraintSet*>& aConstraintSets,
@@ -127,6 +128,7 @@
   const nsString mID;
   const nsString mGroupID;
   const nsString mRawID;
+  const nsString mRawGroupID;
   const nsString mRawName;
 };
 